src/test/java/specs/AroundSpecs.java

```diff
@@ -71,8 +71,7 @@
 
         assertThat(result.getFailureCount(), is(1));
         assertThat(result.getFailures().get(0), is(failure("first spec",
-            RuntimeException.class, "aroundEach did not run the block")));
-
+            RuntimeException.class, "At least one of the test hooks did not run the test block.")));
```

Comment:
Technically a hook could execute an it over multiple examples... so preventing multiple runs might be wrong...but this is not the easiest of frameworks to debug a failing test in. The simpler the hooks the better. 

Let's defer all of this for post merge.